src/app/development/middleware.ts

```diff
@@ -106,6 +106,7 @@ export const templatesDisplayGet: RequestHandler = async (req, res) => {
   const phoneType = context ? (context as string) : undefined;
   /* istanbul ignore else  */
   if (templateId === PAGES.PYI_TRIAGE_DESKTOP_DOWNLOAD_APP) {
+    renderOptions.msBeforeAbort = config.DAD_SPINNER_REQUEST_TIMEOUT;
```

Comment:
```suggestion
    renderOptions.msBeforeAbort = config.DAD_SPINNER_REQUEST_TIMEOUT;
    renderOptions.msBetweenRequests = config.SPINNER_REQUEST_INTERVAL;
    renderOptions.msBeforeInformingOfLongWait =
      config.SPINNER_REQUEST_LONG_WAIT_INTERVAL;
```

Even if not browser-tested yet, it could be worth making this behave the same as the real thing

Reply:
Yes I tested it on my dev env using fake responses.

Reply:
Would it be worth adding in these additional lines?:
```javascript
    renderOptions.msBetweenRequests = config.SPINNER_REQUEST_INTERVAL;
    renderOptions.msBeforeInformingOfLongWait =
      config.SPINNER_REQUEST_LONG_WAIT_INTERVAL;
```